docs/notebooks/channel.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<p>Maybe change to this, with single back ticks around the # and numbers?</p><p> </p><p>where the <code>#</code> is changed to filter wheel <code>1</code> or <code>2.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HinodeXRT/xrtpy/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>